src/pipx/standalone_python.py

```diff
@@ -152,10 +150,10 @@ def get_latest_python_releases() -> List[str]:
         # raise
         raise PipxError(f"Unable to fetch python-build-standalone release data (from {GITHUB_API_URL}).") from e
 
-    return [asset["browser_download_url"] for asset in release_data["assets"]]
+    return [(asset["browser_download_url"], asset.get("digest")) for asset in release_data["assets"]]
```

Comment:
@dukecat0 I used `.get` because it was unclear to me if the API response always has the digest key in it. I mean, it _should_ but this delays the error to be more understandable if there's a problem (at the cost of sprinkling in `| None` in some places

Reply:
Sorry, forgot a `__future__` import